### ISSUE TITLE
Move tutorials to new repo and remove manual

### ISSUE THREAD

**Issue by LoganAMorrison** [LoganAMorrison/Hazma#5]
I think it would be better to isolate the Hazma code and tutorials into separate repositories. It seems like it would be nice to have a standalone `HazmaTutorials` repository which contains all Jupyter notebooks (similar to `DifferentialEquations.jl` and its corresponding `DiffEqTutorials.jl` repository.) 

Also, it doesn't seem like we need the manual. I think the paper + docs should be good enough. @adam-coogan, what do you think?

**Comment by adam-coogan** [comment]
Yeah, that would probably be best. I also agree about not needing a manual -- the paper has a ton of examples.

**Comment by LoganAMorrison** [comment]
Okay, sounds good. I'll work on this when I get a chance.
